pkg/serializer/serializer.go

```diff
@@ -129,6 +131,20 @@ func (w *writer) doSlice(v reflect.Value) {
 	w.byte('}')
 }
 
+func (w *writer) doMap(v reflect.Value) {
+	w.typ(v.Type())
+	w.byte('{')
+	for i, k := range v.MapKeys() {
```

Comment:
Map elements needs to be sorted before serialization. Otherwise this lead to diffs on every re-generations (also will be hard to write tests).